perllib/Open311/UpdatesBase.pm

```diff
@@ -281,7 +281,9 @@ sub _process_update {
     # insertion. We also then need a clone, otherwise the setting of lastupdate
     # will *also* reshift comment->created's time zone to TIME_ZONE.
     my $created = $comment->created->set_time_zone(FixMyStreet->local_time_zone);
-    $p->lastupdate($created->clone);
+    if ($created > $p->lastupdate) {
```

Comment:
I think this works even given the time zone statement above, because both things by here should have a time zone for comparison purpose, I think.